ch5/lkm_template/Makefile

```diff
@@ -187,7 +186,7 @@ checkpatch:
 	@echo
 	@echo "--- kernel code style check with checkpatch.pl ---"
 	@echo
-	$(KDIR)/scripts/checkpatch.pl --no-tree -f --max-line-length=95 *.[ch]
+	$(KDIR)/scripts/checkpatch.pl --no-tree -f --max-line-length=80 *.[ch]
```

Comment:
True, the doc says 80 chars, but I prefer a longer line-length; at times, it's just practical. Hence i keep it as 95 so as not to get those annoying messages..

Reply:
These changes are reverted. 🙏